gtk/gtk.tree-model.lisp

```diff
@@ -989,7 +987,7 @@
  #+cl-cffi-gtk-documentation
  "@version{2013-10-15}
   @argument[path]{a @class{gtk-tree-path} object}
-  @return{@em{True} if @arg{path} has a previous node, and the move was made.}
+  @return{The new @arg{path} if there is a previous node, and the move was made. Otherwise @code{nil}.}
```

Comment:
Well arguably it's not a new path object, so I'd drop the "new" from the sentence?